libminifi/src/utils/crypto/EncryptionManager.cpp

```diff
@@ -32,11 +32,7 @@ namespace minifi {
 namespace utils {
 namespace crypto {
 
-#ifdef WIN32
-constexpr const char* DEFAULT_NIFI_BOOTSTRAP_FILE = "\\conf\\bootstrap.conf";
```

Comment:
Why did you change this?  If we want to change it, should DEFAULT_NIFI_PROPERTIES_FILE be changed similarly, too?  Either way, I don't think this change should be part of this pull request.

Reply:
This was because I changed concat_path to use std::filesystem::path /=, and concatenating "\conf\bootstrap.conf" (an absolute path) to anything results in this second part overwriting the path of the first. This is indeed a behavior change that would ideally not be part of this PR. The reason it is is that I started with this change in the same working directory and I wanted to avoid separating the changes if I could get away leaving them as they are.

Reply:
I would still prefer not separating them, so unless you insist, I would prefer to have these two independent changes reviewed together. Of course this only works if all changes are acceptable.

Reply:
I definitely don't want `DEFAULT_NIFI_PROPERTIES_FILE == "\\conf\\minifi.properties"` and `DEFAULT_NIFI_BOOTSTRAP_FILE == "./conf/bootstrap.conf"` on Windows, so if one of them changes, the other should, too.

I would prefer this change to be part of a separate, `std::filesystem`-related PR, but if it is too much work to separate them, I am going to approve it is at is (after the properties file change).

Reply:
I've removed all filesystem-related changes in the latest rebase